doc(applications-nmap-cli): refresh doc page CTOR-1028 (#4697)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-nmap-cli.md

> Plusieurs articles existent pour vous aider à utiliser ce Pack : 

Reviewer: ```suggestion
Plusieurs articles existent pour vous aider à utiliser ce connecteur : 
```

- [Comment configurer une tâche de découverte ?](https://thewatch.centreon.com/product-how-to-21/discovery-pack-speed-up-your-monitoring-and-make-it-more-reliable-using-the-new-nmap-discovery-tools-149)

> - [Comment améliorer la détection d'équipement et améliorer les propositions de Modèles associés ?](https://thewatch.centreon.com/product-how-to-21/network-discovery-nmap-snmp-how-does-it-work-and-how-can-you-help-162)

Reviewer: ```suggestion
- [Comment améliorer la détection d'équipements et améliorer les propositions de modèles associés ?](https://thewatch.centreon.com/product-how-to-21/network-discovery-nmap-snmp-how-does-it-work-and-how-can-you-help-162)
```

#### Options des modes

Les options disponibles pour chaque modèle de services sont listées ci-dessous :

<Tabs groupId="sync">
<TabItem value="Discovery" label="Discovery">

| Option         | Description                                                                                                                                                                                                                                            |
|:---------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | --subnet       | Specify subnet from which discover resources (mandatory).                                                                                                                                                                 |

Author: ```suggestion
| --subnet       | Specify subnet from which discover resources (must be <ip>/<cidr> format) (mandatory).                                                                                                                                                                |
```

> | --subnet       | Specify subnet from which discover resources (must be <ip>/<cidr> format) (mandatory).                                                                                                                                                                |

Author: ```suggestion
| --subnet       | Specify subnet from which discover resources (must be C<<ip>/<cidr>> format) (mandatory).                                                                                                                                                                |
```

> | --subnet       | Specify subnet from which discover resources (must be <ip>/<cidr> format) (mandatory).                                                                                                                                                                 |

Author: ```suggestion
| --subnet       | Specify subnet from which discover resources (must be <\ip>/<\cidr> format) (mandatory).                                                                                                                                                                 |
```

> | --subnet       | Specify subnet from which discover resources (must be <\ip>/<\cidr> format) (mandatory).                                                                                                                                                                 |

Author: ```suggestion
| --subnet       | Specify subnet from which discover resources (must be C<<\ip>/<cidr>> format) (mandatory).                                                                                                                                                                 |
```

> | --subnet       | Specify subnet from which discover resources (must be C<<ip>/<cidr>> format) (mandatory).                                                                                                                                                                |

Author: ```suggestion
| --subnet       | Specify subnet from which discover resources (mandatory).                                                                                                                                                                |
```

> | --subnet       | Specify subnet from which discover resources (must be \<\ip\>\/<\cidr\> format) (mandatory).                                                                                                                                                                 |

Author: ```suggestion
| --subnet       | Specify subnet from which discover resources (mandatory).                                                                                                                                                                 |
```

> | --nmap-options | Specify the options to use with Nmap. Default value: C<-sS -sU -R -O --osscan-limit --osscan-guess -p U:161,162,T:21-25,80,139,443,3306,8080,8443 -oX - __SUBNET_OPTION__> Note that -oX - are mandatory when using the Plugin in Centreon Host Discovery |

Author: ```suggestion
| --subnet       | Specify subnet from which discover resources (must be \<\ip\>\/<\cidr\> format) (mandatory).                                                                                                                                                                 |
| --nmap-options | Specify the options to use with Nmap. Default value: -sS -sU -R -O --osscan-limit --osscan-guess -p U:161,162,T:21-25,80,139,443,3306,8080,8443 -oX - __SUBNET_OPTION__ Note that -oX - are mandatory when using the Plugin in Centreon Host Discovery |
```

> | --prettify     | Prettify JSON output.                                                                                                                                                                                                                                  |

Author: ```suggestion
| --subnet       | Specify subnet from which discover resources (must be C<<\ip>/<cidr>> format) (mandatory).                                                                                                                                                                 |
| --nmap-options | Specify the options to use with Nmap. Default value: C<-sS -sU -R -O --osscan-limit --osscan-guess -p U:161,162,T:21-25,80,139,443,3306,8080,8443 -oX - __SUBNET_OPTION__> Note that -oX - are mandatory when using the Plugin in Centreon Host Discovery |
| --prettify     | Prettify JSON output.                                                                                                                                                                                                                                  |
```
